settings.py

```diff
@@ -553,6 +553,11 @@
 CELERY_REMINDER_CASE_UPDATE_QUEUE = CELERY_MAIN_QUEUE
 
 
+# This is the celery queue to use for sending repeat records.
+# It's set to the main queue here and can be overridden to put it
+# on its own queue.
+CELERY_REPEAT_RECORD_QUEUE = CELERY_MAIN_QUEUE
```

Comment:
nope not yet, i also still need to update the fab file so that it actually renders the service template conf